tests/network/vmi_multus.go

```diff
@@ -94,6 +94,13 @@ var _ = SIGDescribe("[Serial]Multus", func() {
 		},
 	}
 
+	linuxBridgeInterfaceWithIPAM := v1.Interface{
+		Name: "linux-bridge-with-ipam",
+		InterfaceBindingMethod: v1.InterfaceBindingMethod{
+			Bridge: &v1.InterfaceBridge{},
+		},
```

Comment:
I see nothing in the description to justify the IPAM name.

Reply:
In the future it will be associated with a network that has IPAM.  I can move it closer to the place where this association happens (in the test it self), so that it will be clearer. I can also leave it here and rename it to "linux-bridge-secondary" or something like that, it will be more correct in this context, but less descriptive I think.
Which one do you think is better?

Reply:
It's fine, you can keep it.